packages/LablePicture/package/NewLablePicture/config.ts

```diff
@@ -0,0 +1,4 @@
+export const labelConfig = {
```

Comment:
can you look into config.ts file structure this is required schema needed:
https://github.com/iClasser/iclasser-react-lms-components/blob/develop/packages/MatchingPairs/package/Component/config.ts